### PR TITLE
feat(config): Split config to files

### PR DESCRIPTION
**Pull request by Deniallugo**
## What ❔
Splitting configs to 4 files. 

Genesis -> Params that have been set during Genesis, it’s changing almost every protocol upgrade. but never without smart contract upgrades
contracts -> Unique for each chain and only validator_timelock_addr could be changed during the network lifetime (p.s. we will have an output with all deployed contracts, but it’s not necessary for the server at all)
wallets -> Wallets that we are using for sending stuff receiving stuff, the file should be encrypted.
general -> Server operations could be changed at any time, most of them are safe to change.

## Why ❔

Later on, we will use files instead of env variables and we need to have a proper structure for them

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

File: etc/env/file_based/general.yaml [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+      - prometheus_push_interval_ms: 100
+      - generation_timeout_in_secs: 3600
+      - max_attempts: 5
+      - universal_setup_path: keys/setup/setup_2^26.key
+      - universal_setup_download_url: https://storage.googleapis.com/matterlabs-setup-keys-us/setup-keys/setup_2^26.key
+      - verify_wrapper_proof: true
+  - data_handler:
+      - http_port: 3320
+      - proof_generation_timeout_in_secs: 18000
+      - protocol_version_loading_mode: FromEnvVar
```
[the remark this answers is not in the document]
Comment (Deniallugo) [reply etc/env/file_based/general.yaml:176]: Should we remove it then?

File: etc/env/file_based/general.yaml [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+      - fri_protocol_version_id: 2
+  - group:
```
[the remark this answers is not in the document]
Comment (Deniallugo) [reply etc/env/file_based/general.yaml:178]: Ok, will check

File: etc/env/file_based/general.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Deniallugo) [reply etc/env/file_based/general.yaml:176]: I could just hardcode this value. and do not expose it publicaly

File: etc/env/file_based/general.yaml [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
...111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a,0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6,0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a,0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba,0x92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e,0x4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356,0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97,0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6,0xf214f2b2cd398c806f84e317254e0f0b801d0643303237d97a22a48e01628897,0x701b615bbdfb9de65240bc28bd21bbc0d996645a3dd57e7b12bc2bdf6f192c82,0xa267530f49f8280200edf313ee7af6b827f2a8bce2897751d06a843f644967b1,0x47c99abed3324a2707c28affff1267e45918ec8c3f20b8aa892e8b065d2942dd,0xc526ee95bf44d8fc405a158bb884d9d1238d99f0612e9f33d006bb0789009aaa,0x8166f546bab6da521a8369cab06c5d2b9e46670292d85c875ee9ec20e84ffb61,0xea6c44ac03bff858b476bba40716402b03e41b8e97e276d1baec7c37d42484a0,0x689af8efa8c651a91ad287602527f3af2fe9f6501a7ac4b061667b5a93e037fd,0xde9be858da4a475276426320d5e9262ecfc3ba460bfac56360bfa6c4c28b4ee0,0xdf57089febbacf7ba0bc227dafbffa9fc08a93fdc68e1e42411a14efcf23656e"
```
Comment (montekki) [inline etc/env/file_based/general.yaml:30]: why isn't this a normal yaml array, i think it should be doable

Comment (Deniallugo) [reply etc/env/file_based/general.yaml:30]: Agree, just a copypasta

**Review by montekki — APPROVED** [review]

**Pull request closed without merging**